docs/proposals/20201019-prometheus-rules-in-observatorium-api.md

```diff
@@ -0,0 +1,94 @@
+## Implement API for managing Prometheus Rules
+
+* **Owners:**:
+  * `@squat` / `@ianbillett`
+
+* **Related Tickets:**
+  * `<JIRA, GH Issues>`
+
+* **Other docs:**
+  * `<Links…>`
+
+## TL;DR
+
+We propose implementing an API that allows multi-tenant users to create, read, update and delete Prometheus alerting and recording rules.
```

Comment:
I think it's clearer to say that the API is multi-tenant as opposed to the users
```suggestion
We propose implementing a multi-tenant API that allows users to create, read, update and delete Prometheus alerting and recording rules.
```